docs/v2/openapi.documented.yml

```
@@ -3325,4 +3169,4 @@
   securitySchemes:
     HTTPBearer:
       type: http
-      scheme: bearer
+      scheme: bearer
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)